ota-plus-web/app/reactapp/src/helpers/campaignHelper.js

```diff
@@ -22,46 +22,57 @@ export const getOverallCampaignStatistics = (statistics) => {
   return stats;
 };
 
+const calcRateForStatType = (count, processedCount, percentageFactor) => {
+  if (count === 0) {
+    return 0;
+  }
+  return count / processedCount * 100 < 1
+    ? '< 1'
```

Comment:
Should be moved to translations or at least to const.

Reply:
done